crates/cheatnet/tests/common/mod.rs

```diff
@@ -65,7 +68,113 @@ pub fn deploy_contract(
     let contracts = get_contracts();
 
     let class_hash = blockifier_state.declare(&contract, &contracts).unwrap();
-    deploy(blockifier_state, runtime_state, &class_hash, calldata).unwrap()
+
+    let mut execution_resources = ExecutionResources::default();
+    let mut entry_point_execution_context = EntryPointExecutionContext::new(
+        &build_block_context(runtime_state.cheatnet_state.block_info),
+        &build_transaction_context(),
+        ExecutionMode::Execute,
+        false,
+    )
+    .unwrap();
+    let hints = HashMap::new();
+
+    let mut syscall_hint_processor = SyscallHintProcessor::new(
+        blockifier_state.blockifier_state,
+        &mut execution_resources,
+        &mut entry_point_execution_context,
+        Relocatable {
+            segment_index: 0,
+            offset: 0,
+        },
+        CallEntryPoint::default(),
+        &hints,
+        ReadOnlySegments::default(),
+    );
+
+    deploy(
+        &mut syscall_hint_processor,
+        runtime_state,
+        &class_hash,
+        calldata,
+    )
+    .unwrap()
+}
+
+pub fn deploy_wrapper(
```

Comment:
Added wrapper functions for tests so syscall handlers are constructed automatically (they are not used anywhere in cheatnet tests + cheatnet tests need massive refactors anyways)